### PR TITLE
feat(config): add flag to hide the participant display name

### PR DESCRIPTION
**Pull request by mihhu**
When passed as true, will hide the participant display name from the bottom indicators.

### CODE REVIEW

File: react/features/filmstrip/components/web/Thumbnail.js [hunk 1]
```diff
@@ -230,7 +230,7 @@ const defaultStyles = theme => {
                 marginRight: '4px'
             },
 
-            '&:not(.top-indicators) > *:last-child': {
+            '&:not(.top-indicators) > span:last-child': {
```
Comment (robertpin) [inline react/features/filmstrip/components/web/Thumbnail.js:233]: Why was this changed?

File: react/features/filmstrip/components/web/ThumbnailBottomIndicators.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ import { LAYOUTS } from '../../../video-layout';
 import StatusIndicators from './StatusIndicators';
 
 declare var interfaceConfig: Object;
+declare var config: Object;
```
Comment (robertpin) [inline react/features/filmstrip/components/web/ThumbnailBottomIndicators.js:14]: Don' do this please. Use a selector to get the state from /features/base/config

File: react/features/filmstrip/components/web/ThumbnailBottomIndicators.js [hunk 3]
```diff
@@ -63,21 +64,26 @@ const ThumbnailBottomIndicators = ({
     const styles = useStyles();
     const _allowEditing = !useSelector(isNameReadOnly);
     const _defaultLocalDisplayName = interfaceConfig.DEFAULT_LOCAL_DISPLAY_NAME;
+    const _hideDisplayName = config.hideDisplayName;
```
Comment (robertpin) [inline react/features/filmstrip/components/web/ThumbnailBottomIndicators.js:67]: Use here a selector

**Review by robertpin — CHANGES_REQUESTED** [review]

File: react/features/filmstrip/components/web/Thumbnail.js (hunk 1 above)
Comment (mihhu) [reply react/features/filmstrip/components/web/Thumbnail.js:233]: Because this would've applied on the last status indicator (div) and it would've looked funny. 
image

I saw a lot of logic in the css selectors, are we avoiding classnames by choice?

File: react/features/filmstrip/components/web/ThumbnailBottomIndicators.js (hunk 2 above)
Comment (mihhu) [reply react/features/filmstrip/components/web/ThumbnailBottomIndicators.js:14]: Oh, ok, just followed the interfaceConfig example. Will pass it via props 😃

**Review by robertpin — DISMISSED** [review]

**Review by robertpin — APPROVED** [review]

**Comment by mihhu** [comment]
Please Jenkins test this please.

**Comment by mihhu** [comment]
Please Jenkins test this please.
